File: do-connect-backend/do-connect-backend/src/main/java/com/doconnect/entity/Answer.java
package com.doconnect.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.OneToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "answer")
@Data
@NoArgsConstructor
@AllArgsConstructor
@SuppressWarnings("unused")
public class Answer {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String answer_body;
	private String img_src;
	private boolean approved;
	private String datetime;
	private String approved_by;
	private String created_by;
	@ManyToOne
	@JoinColumn(name="question_id")
	private Question question;
	
}

File: DoConnect-App/src/app/answer/answer.ts
import { Question } from "../question-list/question";

export class Answer {
  
    constructor(
        public id: number,
        public answer_body: string,
        public img_src: string,
        public approved: boolean,
        public datetime: Date,
        public created_by: string,
        public approved_by: string,
        public question: Question
    ) {}
}
